packages/tools/devtools/devtools-view/src/components/data-visualization/TreeHeader.tsx

```diff
@@ -19,23 +20,29 @@ export interface TreeHeaderProps extends HasLabel {
 	 */
 	nodeTypeMetadata?: string | undefined;
 
+	// TODO: metadata
+	metadata?: string | undefined;
+
 	/**
 	 * Inline value to display alongside the metadata.
 	 */
 	inlineValue?: React.ReactElement | string;
 
-	// TODO: metadata
-	metadata?: string | undefined;
+	/**
+	 * TODO
+	 */
+	sharedTreeSchemaData?: string | Record<string | number, string>;
```

Comment:
Same comment as in the visual tree domain - we shouldn't have any dds-specific concepts in this library. I recommend treating this content like a generic tooltip-like thing.